platform-emf/src/main/java/com/softicar/platform/emf/data/table/column/handler/filtering/EmfDataTableColumnFilterNodeFactoryMap.java

```diff
@@ -50,6 +50,7 @@ public EmfDataTableColumnFilterNodeFactoryMap() {
 		addFactory(Integer.class, this::createIntegerFilterNode);
 		addFactory(Long.class, this::createLongFilterNode);
 		addFactory(String.class, this::createStringFilterNode);
+		// TODO PLAT-679 We should add support for IDisplayString here
```

Comment:
This would most likely blow the scope of the story, therefore I added a TODO (should maybe create another story and change the PLAT-Number)